src/test/resources/app_with_menu_display_conditions/suite.xml

```diff
@@ -107,7 +107,6 @@
     </command>
     <instance id="casedb" src="jr://instance/casedb"/>
     <instance id="commcaresession" src="jr://instance/session"/>
-    <instance id="City" src="jr://fixture/item-list:City"/>
```

Comment:
Why are these removed ?

Reply:
I think this was because of a test failure I was getting. This fixture wasn't being referenced anywhere else that I could tell, but I added it back here https://github.com/dimagi/commcare-core/pull/1267/commits/aa5e99d56a1ab8cdaf50d87400f3b28ea1579e81. This commit https://github.com/dimagi/commcare-core/pull/1267/commits/338f5f012e7808784e50346ed9662dadd91a710e fixed the issue by properly limiting instances.